server/controllers/auth.js

```diff
@@ -63,7 +63,7 @@ exports.loginUser = asyncHandler(async (req, res, next) => {
     });
 
     if (user.boards.length === 0) {
-        user.boards = await generateBoard();
+        user.boards = [ await generateBoard('My School Board') ];
         await User.findByIdAndUpdate(user._id, user ).exec();
     }
```

Comment:
If a user has no boards intentionally do we want to create them a brand new board? I think this would not be necessary because what if as a user I wanted to have no boards?

Reply:
This was added back when board creation hasn't been implemented yet for users created before multiple boards were added. I'm not sure what to do about the lack of boards yet (e.g. display a notification in the board area) but I will remove this section.